src/odemis/acq/test/fastem_test.py

```diff
@@ -439,7 +439,7 @@ def setUpClass(cls):
         cls.multibeam = model.getComponent(role="multibeam")
         cls.descanner = model.getComponent(role="descanner")
         cls.stage = model.getComponent(
-            role="stage")  # TODO replace with stage-scan when ROA conversion method available
+            role="stage-scan")  # TODO replace with stage-scan when ROA conversion method available
```

Comment:
That change also shouldn't be here. Maybe it's a useful change too though... and you'd need to drop the TODO.

Reply:
I accidentally committed this,  unfortunately it's a bit more work to fix this, it will also need to be replaced and tested in the acquisition code.